airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/CreateConnectorModal.tsx

```diff
@@ -219,7 +235,7 @@ const CreateConnectorModal: React.FC<IProps> = ({ onClose, onSubmit, errorMessag
                   >
                     <FormattedMessage id="form.cancel" />
                   </Button>
-                  <Button type="submit" disabled={isSubmitting || !dirty || !isValid} isLoading={isSubmitting}>
+                  <Button type="submit" disabled={isSubmitting || !dirty} isLoading={isSubmitting}>
```

Comment:
Any specific reason to remove `!isValid` as a reason to disable submission? AFAIK we usually disable submission buttons if the form is not valid (but I know this is not 100% consistent).

Reply:
It's not entirely consistent and I _think_ we were moving away from disabling the button and towards having more consistent and useful validation?  But I've turned it back on for now as that wasn't part of this ticket and instead added "Required" labels to the required fields.

<img width="949" alt="Screenshot 2023-01-09 at 9 47 59 AM" src="https://user-images.githubusercontent.com/63751206/211335738-1fb5fb77-9e09-4f40-9b06-bf1f7a471de2.png">

There's a funky state when a user tabs into the URL space and starts typing (the button will re-disable until the URL is valid and a user won't see the error message until they tab or click out of that field) but I don't feel too strongly on it.

Reply:
How about displaying the error message for the docs URL regardless of `meta.touched`? I think that's slightly nicer UX, since it's an optional field that can only become invalid once you start typing.

Reply:
👍 that'll work.  Added in https://github.com/airbytehq/airbyte/pull/21034/commits/f9fb974b513548a54dd3b7e3153ca1ac0a0e79bc